### ISSUE TITLE
There is no way to supply image pull secrets

### ISSUE THREAD

**Issue by philgebhardt** [gremlin/helm#43]
As a Gremlin user, when my cluster uses a private registry that is protected by a pull secret, I want a way to pass this secret to the Gremlin Helm chart, so that I may pull down images from my private repository without having to modify Gremlin Helm charts.

This might looks something like:
```
--set image.pullSecret=$PULLSECRET --set chaoimage.pullSecret=$PULLSECRET
```

**Comment by philgebhardt** [comment]
This is now available in helm chart `0.4.2` (#47)

**Comment by markshields-gl** [comment]
Thanks @philgebhardt !
